test/acceptance/view_benchmark_graphs_test.rb

```diff
@@ -90,6 +141,35 @@ class ViewBenchmarkGraphsTest < AcceptanceTest
     end
   end
 
+  test "User should be able to view long running benchmark graphs after they select different number of benchmark 
+  runs, when they go back and forward in the broswer window" do
+
+    javascript_driver = Capybara.javascript_driver
+    default_driver = Capybara.current_driver
+    Capybara.javascript_driver = :selenium
+    Capybara.current_driver = :selenium
+
+    benchmark_type = benchmark_types(:array_count)
```

Comment:
Nitpick: Add a blank line before and after for code blocks. Generally this makes it easier to read.